app/src/main/java/com/codingblocks/cbonlineapp/mycourse/player/VideoPlayerActivity.kt

```diff
@@ -786,5 +887,11 @@ class VideoPlayerActivity : BaseCBActivity(), EditNoteClickListener, AnkoLogger,
                 VIDEO_POSITION to position,
                 SECTION_ID to sectionId).singleTop()
         }
+
+        fun createVideoPlayerActivity(context: Context, contentId: String, sectionId: String, exclude: Boolean): Intent {
```

Comment:
also you can use the exisiting method in that case 
https://github.com/coding-blocks/CBOnlineApp/blob/688f7fdfe99e5cce1f4f2dd120a1552686a62ed8/app/src/main/java/com/codingblocks/cbonlineapp/mycourse/player/VideoPlayerActivity.kt#L884